src/nvim/move.c

```diff
@@ -74,8 +71,9 @@ static void comp_botline(win_T *wp)
   }
 
   for (; lnum <= wp->w_buffer->b_ml.ml_line_count; ++lnum) {
-    last = lnum;
-    folded = FALSE;
+    int n;
+    linenr_T last = lnum;
+    int folded = FALSE;
```

Comment:
You can’t add line with FALSE without failing the clint build. Or touch any line near line with FALSE as well.